openfish-webapp/README.md

```diff
@@ -0,0 +1,39 @@
+# Openfish Webapp
+Openfish Webapp is a web interface for interacting with the Openfish API.
+
+## Development guide
+#### Getting started
+
+Have the following prerequisites installed on your system:
```

Comment:
Given that you have a Dockerfile now, I would put instructions for running with that before this.

Reply:
Yep, done